Gopkg.toml

```diff
@@ -55,11 +55,11 @@
 
```

Comment:
Should we pin the velero to a specific version?

Reply:
we can definitely do that just after velero 1.4 is out, before releasing a new version of this plugin. Until then, there is no version of velero that has the changes that we want to integrate/test with.

Reply:
Sounds good!
I will also convert this to switch to go modules for all the plugins.

Reply:
cool, yeah also wanted to do that but didn't want to take the time right now. I'm planning to create issues in all the plugin repos for (a) go module migration, and (b) refactoring the repo org/makefile/dockerfile to be closer to what's in the example repo